packages/core/src/routes/sign-in-experience.ts

```diff
@@ -42,7 +43,14 @@ export default function signInExperiencesRoutes<T extends AuthedRouter>(router:
     }),
     async (ctx, next) => {
       const { body } = ctx.guard;
+      const { branding, termsOfUse } = body;
 
+      validateBranding(branding);
+      validateTermsOfUse(termsOfUse);
+      // TODO: validate SignInMethods
+      // TODO: validate socialConnectorIds
+
+      // TODO: Only update socialSignInConnectorIds when social sign-in is enabled.
       ctx.body = await updateDefaultSignInExperience({
```

Comment:
Is there any other method named 'updateSignInExperience'?

Reply:
No. There are only one update function for sign-in experience for now.